mongo/bboard_v1.json

```diff
@@ -0,0 +1,76 @@
+[
+    {
+        "//": "Comment: bountyHash message -> bankless dao bounty board"
+    },
+    {
+        "season" : 2,
+        "bountyTitle": "A new About Us webpage",
+        "bountyDescription": "Description of webpage",
+        "doneCriteria": "A completed bounty has the following...",
```

Comment:
we could change to bountyCriteria for consistency of field names

Reply:
Sounds good, changes made, will push.